teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">&nbsp;&nbsp;# Store keys of training and test sets
&nbsp;&nbsp;train_keys = train[['SMILES', 'accession']]
&nbsp;&nbsp;test_keys = test[['SMILES', 'accession', 'pchembl_value_Mean']].reset_index(drop=True)

</pre><p><code>keys</code> makes me think of a dict - but we have DataFrames in the variables, is that right?</p><p>Can we do an index reset in both cases for consistency or do the indices in <code>train_keys</code> mean something?</p><p>--</p><p>PEP8 wants <a href="https://peps.python.org/pep-0008/#function-and-variable-names" target="_blank">lower case variables</a> (I know we have not respected this in T007 but should have)</p><p>model_RF &gt; model_rf </p><p>-- </p><p>I will stop with my review for now here and will continue tomorrow the latest. If you want to get started right away, feel free to do so!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
<p>Keys in this case are needed to link back the resulted predictions to their corresponding compound-target pair, so we can calculate e.g. performance per target. Identifiers could work better? </p>